Crossref Unixref XML.js

```diff
@@ -44,6 +44,13 @@
 /** ********************
  * Utilitiy Functions *
  **********************/
+const datasetType = ZU.fieldIsValidForType('title', 'dataset')
+	? 'preprint'
+	: 'document';
+const standardType = ZU.fieldIsValidForType('title', 'standard')
+	? 'preprint'
+	: 'report';
```

Comment:
Right, I somehow had it in my head these were only in the beta, but they're in stable since last April. (2. was a typo, no longer relevant either way)